src/dodal/common/coordination.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+
+
+def group_uuid(name: str) -> str:
+    """
+    Returns a unique but human-readable string, to assist debugging orchestrated groups.
+
+    Args:
+        name (str): A human readable name
+
+    Returns:
+        readable_uid (str): name appended with a unique string
+    """
+    return f"{name}-{str(uuid.uuid4())[:6]}"
+
+
+def inject(name: str):  # type: ignore
+    """
+    Function to mark a default argument of a plan method as a reference to a device
+    that is stored in the Blueapi context.
+    Bypasses mypy linting, returning x as Any and therefore valid as a default
+    argument.
```

Comment:
Should: This looks very weird as a function that's like a noop. Could you give an example in the docstrings in how to use it?

Reply:
Added more detail